### DISCUSSION TITLE
GPU support for Milvus cluster

### DISCUSSION THREAD

**Discussion by ArijitSinghEDA** [milvus-io/milvus#26600] [Q&A and General discussion]
Does Milvus Cluster enable GPU by default? Or is GPU irrelevant in case of Cluster?
I have very little idea about architecture in general.

**Comment by xiaofan-luan** [comment]
if you want to setup GPU, maybe look at this document
https://milvus.io/docs/install_standalone-gpu-docker.md

**Reply by ArijitSinghEDA** [reply]
Actually my doubts are regrading Milvus Cluster. If I deploy it, will it use existing GPUs of connected servers?

**Comment by pancodia** [comment]
I followed this doc to install  milvus standalone with GPU enabled. How can I verify if the deployment is correct and GPU is used?

**Reply by ponponon** [reply]
用 nv 提供的工具看显存和核心的使用率
